From 2c083af3d06d333db31dcc1bbe535b10285a8d1e Mon Sep 17 00:00:00 2001
From: "Michael S. Klishin" <michael@novemberain.com>
Date: Sun, 23 May 2010 21:09:03 +0400
Subject: [PATCH] Makes HTTPS client tests pass

https://www.amazon.com now redirects to http version. Switching site
is enough to make the test pass.

Signed-off-by: Aman Gupta <aman@tmm1.net>
---
 tests/test_httpclient2.rb |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/tests/test_httpclient2.rb b/tests/test_httpclient2.rb
index ef72e7c..4705e42 100644
--- a/tests/test_httpclient2.rb
+++ b/tests/test_httpclient2.rb
@@ -141,7 +141,7 @@ class TestHttpClient2 < Test::Unit::TestCase
   def test_https_get
     d = nil
     EM.run {
-      http = EM::P::HttpClient2.connect :host => 'www.amazon.com', :port => 443, :ssl => true
+      http = EM::P::HttpClient2.connect :host => 'www.apple.com', :port => 443, :ssl => true
       d = http.get "/"
       d.callback {
         EM.stop
-- 
1.7.7.5

